### PR TITLE
Updating Serbian translation

### PR DESCRIPTION
**Pull request by alexej996**
As per issue https://github.com/monero-project/monero-gui/issues/1377

### CODE REVIEW

**Comment by erciccione** [comment]
Perfect. Thank you :)

**Comment by xmronadaily** [comment]
The updated translation is great, just a small typo at line 464:   <translation>Deljena RIngDB</translation> just lower case i there.

File: translations/monero-core_sr.ts [hunk 1]
```diff
@@ -547,549 +537,605 @@
...
-        <source>Wallet creation height: </source>
-        <translation>Visina u blokovima za kreiranje novčanika:</translation>
-    </message>
-    <message>
-        <location filename="../pages/Settings.qml" line="498"/>
-        <source> &lt;a href=&apos;#&apos;&gt;(Click to change)&lt;/a&gt;</source>
-        <translation> &lt;a href=&apos;#&apos;&gt;(Klikni da promeniš)&lt;/a&gt;</translation>
+        <location filename="../pages/Settings.qml" line="657"/>
+        <source>Wallet name: </source>
+        <translation>Naziv novčanika:</translation>
```
Comment (erciccione) [inline translations/monero-core_sr.ts]: please add a space after `:` as in the source text

File: translations/monero-core_sr.ts [hunk 2]
```diff
@@ -547,549 +537,605 @@
...
     </message>
     <message>
-        <location filename="../pages/Settings.qml" line="524"/>
-        <source>Save</source>
-        <translation>Sačuvaj</translation>
+        <location filename="../pages/Settings.qml" line="671"/>
+        <source>Wallet creation height: </source>
+        <translation>Visina u blokovima za kreiranje novčanika:</translation>
```
Comment (erciccione) [inline translations/monero-core_sr.ts]: please add a space after `:` as in the source text

File: translations/monero-core_sr.ts [hunk 3]
```diff
@@ -547,549 +537,605 @@
...
         <source>Please enter wallet password</source>
         <translation>Unesite lozinku novčanika</translation>
     </message>
     <message>
-        <location filename="../components/PasswordDialog.qml" line="87"/>
-        <source>Please enter wallet password for:&lt;br&gt;</source>
-        <translation>Unesite lozinku novčanika za:&lt;br&gt;</translation>
+        <location filename="../components/PasswordDialog.qml" line="86"/>
+        <source>Please enter wallet password for: </source>
+        <translation>Unesite lozinku novčanika za:</translation>
```
Comment (erciccione) [inline translations/monero-core_sr.ts]: please add a space after `:` as in the source text

**Pull request merged**